Hi3Helper.Core/Lang/en_US.json

```diff
@@ -635,7 +637,20 @@
     "FileDownloadSettings_BurstDownloadHelp4": "features to run in parallel to make the download process more efficient.",
     "FileDownloadSettings_BurstDownloadHelp5": "When disabled, the download process for",
     "FileDownloadSettings_BurstDownloadHelp6": "and",
-    "FileDownloadSettings_BurstDownloadHelp7": "features will use a sequential download process."
+    "FileDownloadSettings_BurstDownloadHelp7": "features will use a sequential download process.",
+    
+    "Database_Title": "User Synchronizable Database",
+    "Database_ConnectionOk": "Database connected successfully!",
+    "Database_ConnectFail": "Failed to connect to the database, see error(s) below:",
+    "Database_Toggle": "Enable Online Database",
+    "Database_Url": "Database URL",
+    "Database_Url_Example": "Example: https://db-bagusnl.turso.io",

```

Comment:
`Examples: https://db-collapse.turso.io` so that it represents the project

Reply:
Fixed